components/nds-full-bleed/README.md

```diff
@@ -0,0 +1,107 @@
+# `@nice-digital/nds-full-bleed`
+
+> Full bleed (edge-to-edge viewport) element for the NICE Design System
+
+- [`@nice-digital/nds-full-bleed`](#nice-digitalfullbleed)
+- [Installation](#installation)
+- [Usage](#usage)
+- [React](#react)
+- [Props](#props)
+- [SCSS](#scss)
+- [HTML](#html)
+
+## Installation
+
+Install Node, and then:
+
+```sh
+npm i @nice-digital/nds-full-bleed
+```
+
+## Usage
+
+### React
+
+Import the `FullBleed` component from the package and use with JSX:
+
+```jsx
+import { FullBleed } from "@nice-digital/nds-full-bleed";
+<FullBleed>
+	<p>Content here</p>
+</FullBleed>
+```
+
+The component doesn't have any internal spacing or structure, and any supplied markup will align to the left of the viewport by default. Include structural markup if you need to, for example:
+
+```jsx
+import { FullBleed } from "@nice-digital/nds-full-bleed";
+import { Grid, GridItem } from "@nice-digital/nds-grid";
+<FullBleed>
+	<div className="container">
+		<Grid>
+			<GridItem cols={12}>
+				<p>This content is aligned to the grid</p>
+			</GridItem>
+		</Grid>
+</FullBleed>
+```
+
+### Props
```

Comment:
Is it worth mentioning that any other props are spread over the containing div like we do for other components?